### ISSUE TITLE
Disclaimer option to cancel (Scenario: User not agree with disclaimer text)

### ISSUE THREAD

**Issue by creativeanalyst** [globaleaks/GlobaLeaks#2956]
Currently Disclaimer pop-up message window has only one option 'Proceed' after reviewing disclaimer text. It looks we need an 'Cancel' option for user to select if user is not agreeing to disclaimer text and doesn't want to proceed.

**Comment by evilaliv3** [comment]
Thank you @creativeanalyst 

I consider that we could just add a close button, what do you think ?

**Comment by creativeanalyst** [comment]
Agree. Close or Cancel or Disagree button basically to close pop-up disclaimer screen and take back to previous page.
